python/artm/artm_model.py

```diff
@@ -582,10 +574,10 @@ def _func(batch):
 
         self._wait_for_batches_processed(
             self._pool.apply_async(func=self.master.fit_online,
-                                   args=(batches_list, batch_vectorizer.weights,
+                                   args=(batch_vectorizer.batches_ids, batch_vectorizer.weights,
                                          update_after_final, apply_weight_final,
                                          decay_weight_final, async)),
-            len(batches_list))
+            len(batch_vectorizer.batches_ids))
```

Comment:
I agree, that len(...) should be replaces with method call, but I don't think, that `__len__` is a good idea, because `num_batches` is more convenient.